src/syft/grid/connections/http_connection.py

```diff
@@ -0,0 +1,104 @@
+# third party
+import requests
+
+# syft absolute
+import syft as sy
+from syft.core.common.message import SignedImmediateSyftMessageWithReply
+from syft.core.common.message import SignedImmediateSyftMessageWithoutReply
+from syft.core.common.message import SyftMessage
+from syft.core.io.connection import ClientConnection
+from syft.decorators.syft_decorator_impl import syft_decorator
+
+
+class HTTPConnection(ClientConnection):
+    @syft_decorator(typechecking=True)
+    def __init__(self, url: str) -> None:
+        self.base_url = url
+
+    @syft_decorator(typechecking=True)
+    def send_immediate_msg_with_reply(
+        self, msg: SignedImmediateSyftMessageWithReply
+    ) -> SignedImmediateSyftMessageWithoutReply:
+        """Sends high priority messages and wait for their responses.
+
+        NOTE: Mandatory abstract method.
```

Comment:
Q: Why not mark it with the ```abstractmethod``` decorator?

Reply:
Do you mean, apply it at the `core.io.connection.ClientConnection` superclass?

Reply:
If I understand correctly, @IonesioJunior is saying that these methods are required by the parent class ClientConnection, even though their implementation is basically always the same in this case? If that is the meaning then, I think we could possibly just remove the wording "Note: Mandatory abstract method".

I have made this change and some other small tweaks and fixes to the comments.